lib/src/formula.dart

```diff
@@ -20,11 +20,6 @@ String updateFormula(String channel, String contents, String version,
       r'/(\d[\w\d\-\.]*)/sdk/([\w\d\-\.]+)\"\n(\s+)sha256 \"[\da-f]+\"',
       multiLine: true);
   return contents.replaceAllMapped(filesAndHashes, (m) {
-    var currentVersion = m.group(1);
```

Comment:
Remove group(1) from the regular expression, it's no longer needed and can easily be taken out by removing the parenthesis and updating the other group numbers.

Reply:
Done